### DISCUSSION TITLE
Open-source real life examples with KeystoneJS

### DISCUSSION THREAD

**Discussion by daawaan4x** [keystonejs/keystone#7486] [Questions] [answered]
Hello everyone, I've been playing around with KeystoneJS but the docs don't really dive deep into making a real life application with KeystoneJS. Are there any examples available that uses KeystoneJS in a real life production setting?

**Comment by borisno2** [comment, marked as answer]
Hello @daawaan,
The biggest example in the keystone repo is the one from the Wes Bos Advanced React course [here](https://github.com/keystonejs/keystone/tree/main/examples-staging/ecommerce). 

I have open sourced one that I am using for club memberships using stripe for subscription payments. you can find that here https://github.com/OpenSaasAU/clubhouse I am currently using this in production, but still something I am iterating on.

Hope this helps.

**Comment by wchorski** [comment]
I learned off the Wes Bos Adv React Course for this repo, but went way beyond (using the keystone inside of NextJS app framework)

Right now the most 'up to date' repo is inside a branch but hoping to merge it with main branch sometime this year

https://github.com/wchorski/KYPN/tree/add-all-schemas

**Comment by pahaz** [comment]
Hi there! 👋

We’re maintining an open-source Property Management SaaS built on KeystoneJS in production and serving 4 000+ clients with tickets, invoicing, payments, reporting and so. We have some contributors from around the globe, and shows Keystone in real-world apps.

Feel free to explore the code here: https://github.com/open-condo-software/condo

If you find it useful, we’d be thrilled if you dropped us a ⭐. Happy building! ✨
